### DISCUSSION TITLE
toktx doesn't compress images ?

### DISCUSSION THREAD

**Discussion by kamil-s-kom** [KhronosGroup/KTX-Software#661] [Q&A] [answered]
I tried to use toktx like this
`toktx --encode ect1s logo.ktx2 logo-square.png`
but the resulting file doesn't seems to be compressed. It has 2 mb size while the original png is only 46 kb.

I use ktxinfo to check the generated file and it says that the super compression is KTX_SS_NONE. I thought by specifying the encoding I will have some sort of default compression.

**Comment by MarkCallow** [comment, marked as answer]
You have a typo `ect1s` instead of `etc1s`. `toktx `is failing to report the unknown encoder and acting as if no encode option was set. That is, no compression is done.

There are many reasons why an uncompressed image could end up larger than its PNG file.
